### DISCUSSION TITLE
svh, dvh, lvh

### DISCUSSION THREAD

**Discussion by amitu** [fastn-stack/fastn#1608] [Ideas & RFCs]
https://www.youtube.com/watch?v=ru3U8MHbFFI

**Comment by amitu** [comment]
@ishaan701 has started working on this: https://github.com/fastn-stack/fastn/pull/1631.

**Comment by Arpita-Jaiswal** [comment]
Implemented in https://github.com/fastn-stack/fastn/pull/1631.
